docs/project-backlog.md

```diff
@@ -0,0 +1,122 @@
+# Backlog Structure for OpenTelemetry
+
+It's an important community goal for OpenTelemetry that our members find the backlogs to be responsive, and easy to take part in. Having a shared methodology for processing issues helps new members understand the current state of each project.
+
+## Roles
+* **Triager**: Person who is triaging the issue by determining its workability. This person is responsible for getting the ticket to one of two stages - 1) ready-for-work 2) wontfix. They are responsible for triaging  by working with the OP to get additional information as needed and analyzing the issue and adding relevant details/information/guidance that would be helpful to the resolution of the issue.
```

Comment:
There is no formal `Triager` in community membership document. My first reaction is that we should just make it maintainer's responsibility. However I can see cases when someone helping maintainers with triage. If you envision this situation - it will be good to mention it in community membership document